docs/appeng/adrs/0006-web-development-framework-preference.md

```diff
@@ -0,0 +1,87 @@
+- **Accessibility**: Uses Python which is considered easy to learn and suitable for juniors, strong community support, robust documentation.
+
+#### Spring Boot (Java)
+
+- **Pros**: Mature, enterprise-grade, scalable, good for microservices, static typing.
+- **Cons**: Steeper learning curve, more configuration required, less suitable for rapid prototyping.
+- **Accessibility**: Uses Java which is commonly taught in college programs, although notably more verbose than Python and Javascript.
+
+#### Node + Express.js (Typescript)
+
+- **Pros**: Mature, high flexibility, large ecosystem, strong community support, suitable for real-time applications.
+- **Cons**: Requires more boilerplate, less guidance on structure.
+- **Accessibility**: If we're abiding by prior ADRs it would mean using Typescript, which may have some learning curve for folks coming in without that background.
+
+### Front-End Frameworks
+
+#### React.js
+
+- **Pros**: Industry standard, high flexibility, strong community, large ecosystem, easily build cross-platform apps with React Native.
+- **Cons**: JSX learning curve, more choices for state management, constant updates require developers to relearn newer concepts to keep up.
+- **Accessibility**: Good for juniors with Javascript basics.
+
+#### Angular
+
+- **Pros**: Lots of tools provided out of the box, strong community, long-term maintainability since it’s backed by Google.
+- **Cons**: No strong support for cross-platform apps.
+- **Accessibility**: Requires skill in Typescript, which may have some learning curve for folks coming in without that background.
+
+#### Vue.js
+
+- **Pros**: Focuses on helping beginner developers create dynamic web apps without prior tedious learning.
+- **Cons**: Smaller community, no strong support for cross-platform apps, doesn’t render in older versions of web browsers.
+- **Accessibility**: Known for easy learning curve.
+
+### Cloud Infrastructure Providers
+
+#### Amazon Web Services (AWS)
+
+- **Pros**: Market leader in cloud services, suitable for small and enterprise scale, extensive documentation, large community and support, has the largest number of data centers.
+- **Cons**: High learning curve, naming conventions tend to be confusing, user interface is not good.
+- **Accessibility**: Amazon provides excellent online training resources for all levels.
+
+#### Google Cloud Platform (GCP)
+
+- **Pros**: Excels in big data processing and ML, leading in Kubernetes and open-source integrations, focuses on high-performance networking and low-latency connections.
+- **Cons**: Fewer resources and community support compared to AWS.
+- **Accessibility**: Slightly more accessible due to focus on open-source and user-friendly interfaces.
+
+#### Microsoft Azure
+
+- **Pros**: Robust offerings in AI and ML due to Microsoft’s partnership with OpenAI, integrates with Microsoft ecosystem.
+- **Cons**: High learning curve, not as good customer service.
```

Comment:
security is also usually lacking, or at least, has been historically with Azure.